website/plasma-web-docs/package.json

```diff
@@ -14,39 +14,50 @@
     "access": "restricted"
   },
   "scripts": {
-    "NODE_OPTIONS": "NODE_OPTIONS=--openssl-legacy-provider",
-    "docusaurus": "docusaurus",
-    "start": "npm run NODE_OPTIONS && docusaurus start",
-    "build": "npm run NODE_OPTIONS && docusaurus build",
-    "swizzle": "npm run NODE_OPTIONS && docusaurus swizzle",
-    "deploy": "npm run NODE_OPTIONS && docusaurus deploy",
-    "clear": "npm run NODE_OPTIONS && docusaurus clear",
-    "serve": "npm run NODE_OPTIONS && docusaurus serve",
-    "write-translations": "npm run NODE_OPTIONS && docusaurus write-translations",
-    "write-heading-ids": "npm run NODE_OPTIONS && docusaurus write-heading-ids",
+    "start": "docusaurus start",
+    "start:dev": "npm run start --dev --no-minify",
```

Comment:
https://docusaurus.io/docs/cli#docusaurus-cli-commands

- `--no-minify` Build website without minimizing JS/CSS bundles.
- `dev` Builds in dev mode, including full React error messages.